### DISCUSSION TITLE
how to embed a linear synteny view in my website

### DISCUSSION THREAD

**Discussion by Zoe81777** [GMOD/jbrowse-components#3658] [Q&A]
Dear author,
  I have embedded a JBrowse 2 linear genome view in my website using the [tutorial](https://jbrowse.org/jb2/docs/tutorials/embed_linear_genome_view/01_introduction/), which is very convenient. Now, I have problem in embedding a linear synteny view. I found no tutorial for embedding linear synteny view in my website. Please help me.

**Comment by cmdcolin** [comment]
Hi there,
The synteny view is not currently embeddable. I'd recommend setting up the whole jbrowse-web application, which contains the synteny view https://jbrowse.org/jb2/docs/quickstart_web/
